common/theme.c

```diff
@@ -46,23 +54,20 @@ void themeStartup(ThemePreset preset) {
         .hbmenuLogoImage = hbmenu_logo_dark_bin
     };
     
-    char tmp_path[PATH_MAX] = {0};
-
-    #ifdef __SWITCH__
-    tmp_path[0] = '/';
-    #endif
-
-    strncat(tmp_path, "config/nx-hbmenu/themes/theme.cfg", sizeof(tmp_path)-2);
     
+    const char* themePath = "";
+    GetThemePathFromConfig(themePath);
+    
+
     theme_t *themeDefault;
     config_t cfg = {0};
     config_init(&cfg);
     config_setting_t *theme;
-    color_t text, frontWave, middleWave, backWave, background, highlight, separator;
+    color_t text, frontWave, middleWave, backWave, background, highlight, separator, borderColor, borderTextColor;
     int waveBlending;
     const char *AText, *BText;
-    bool good_cfg = config_read_file(&cfg, tmp_path);
-    
+    bool good_cfg = config_read_file(&cfg, themePath);
```

Comment:
Check for ```themePath[0]==0``` and set ```good_cfg = false```, otherwise run config_read_file.

Reply:
shouldnt the initialization ="" take care of this? i have tested this and it seems to work fine...

Reply:
Would prefer if it wouldn't attempt reading a file with path "".